docker-compose.yaml

```diff
@@ -50,3 +50,15 @@ services:
       build:
         context: .
         dockerfile: splinterd/api/openapi.dockerfile
+
```

Comment:
typo in commit message Spolinter -> Splinter

Reply:
resolved